scripts/collect_workloads.py

```
@@ -727,7 +756,12 @@ def run_sglang_mode(
             str(tp),
             "--attention-backend",
             "flashinfer",
-            "--disable-cuda-graph",
+            # NOTE: cuda graphs are intentionally enabled. Level-10 dumps
+            # work under graph capture/replay because flashinfer auto-
+            # patches torch.cuda.CUDAGraph.replay (in every process that
+            # imports flashinfer with FLASHINFER_LOGLEVEL>=10) to call
+            # flush_graph_dumps() after each replay. Without graphs,
+            # collection takes 20-50× longer in eager mode.
```

Ruff (0.15.11)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)